process_report/invoices/bu_internal_invoice.py

```diff
@@ -15,7 +15,11 @@ class BUInternalInvoice(invoice.Invoice):
         invoice.INVOICE_DATE_FIELD,
         invoice.PI_FIELD,
         "Project",
```

Comment:
Not related to this PR, but you should switch this field (and all other occurences of the magic string "Project") to instead using `invoice.PROJECT_NAME_FIELD` in a future patch.